draft-ietf-tls-tls13.md

```diff
@@ -2124,24 +2127,31 @@ mode, the server MUST send Certificate ({{certificate}}) and CertificateVerify
 {:/comment}
 {:br}
 
-In order to accept PSK key establishment, the server sends a
-"pre_shared_key" extension with the selected identity.
+Each PSK MUST be associated with a single Hash algorithm. For PSKs established
+via the ticket mechanism ({{NewSessionTicket}}), this is the Hash used for
+the KDF. For externally established PSKs, the Hash algorithm is set when the
+PSK is established.
```

Comment:
My impiression was that the entity creating the PSK would dictate a single hash algorithm to use with it.  (For pure-PSK, that is.)